src/Redux/Books/Books.js

```diff
@@ -0,0 +1,28 @@
+const ADD = 'BookStore/Books/ADD_BOOK';
+const REMOVE = 'BookStore/Books/REMOVE_BOOK';
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD:
+      return {
+        ...state,
+        books: [state.title, state.author, action.type],
+      };
+    case REMOVE:
+      return {
+        ...state,
+        books: state.filter((book) => book.title !== action.type),
+      };
```

Comment:
Creates and Exports **Action Creators**. (`addBook` and `removeBook` functions) 📝 Good job!